### DISCUSSION TITLE
Font size in article list

### DISCUSSION THREAD

**Discussion by sozifa** [jocmp/capyreader#209] [Feature Requests]
### Steps to Reproduce

Many thanks for adding adjustable font solize in article view, - would it be possible to have thatvoption for the article list as well -many thanks again 

### Expected behavior

Font size in article viw adjustable

**Comment by jocmp** [comment]
@sozifa glad it's working for you! I like the idea of having that option. What part of the text do you want to change, just the title or also the time and feed name?

**Reply by sozifa** [reply]
for me just the title would be fine but that might be different for others...
In general I like options ;-) - so if the article list would  have setting options others than just two thumbnail sizes that would be great (for me only font size and a mid thumbnail size would be perfect - though moving the thumbnails to the left might also be something...)

**Comment by sozifa** [comment]
Did you find the time to give this another thought? Would really appreciate it - always need my glasses to rear the article list, what isn't that handy for a quick news update on the go ;-)

**Reply by jocmp** [reply]
hey @sozifa - sorry for the small font. this is still on my list! I should get to this after I wrap up https://github.com/jocmp/capyreader/pull/297 or #91.

**Reply by jocmp** [reply]
@sozifa - I added a font scale option in the latest release, [2024.09.1040](https://github.com/jocmp/capyreader/releases/tag/2024.09.1040). It's found under Settings > Display & Appearance > Font Scale

<img src="https://github.com/user-attachments/assets/e5804c7a-3ab2-4a76-a400-6a61dfc3e126" width="500px">

**Reply by sozifa** [reply]
Many thanks, really appreciated

**Reply by devguy23857848467** [reply]
Would it be possible to make this font choice more granular (ex. more steps on the scale)?
I have the base system font set to largest size, and find that the font sizing in Cappy Reader is too extreme. 
I currently have it on the 4th step... I would like a setting somewhere between the 4th and 5th steps...
FeedFlow has this feature, if that helps...

**Comment by jocmp** [comment]
Closing as done.
